patches/rlz-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/rlz/BUILD.gn b/rlz/BUILD.gn
+index 37247ec1e5683d6124335bb1f6405618169283a4..371f42d8ce0fc86f7946615b5119fcce84071565 100644
+--- a/rlz/BUILD.gn
++++ b/rlz/BUILD.gn
+@@ -208,6 +208,7 @@ if (enable_rlz_support) {
+     }
+   }
+ }
++else { group("rlz_lib") {} group("rlz_lib_no_network") {} group("rlz_unittests") {} }
```

Comment:
this patch and the existing patch for `enable_rlz_support` can be removed because `enable_rlz` is false by default `enable_rlz = is_chrome_branded && enable_rlz_support`

Reply:
Removed patches in 6337990115611141196d144b07b63551bf9a4a03